README.md

```diff
@@ -85,11 +85,11 @@ After you created the component structure folder, you wrote unit test, you wrote
 Nice, so you have already created a new component :+1:, that is great!
 It's time to check if everything works as expected.
 
-*If you want to **locally test a component from within your project**. See [How to locally test blueprint library in your project](#how-to-locally-test-blueprint-library-in-your-project) Otherwise, read on to test your built component in blueprint.*
+_If you want to **locally test a component from within your project**. See [How to locally test blueprint library in your project](#how-to-locally-test-blueprint-library-in-your-project) Otherwise, read on to test your built component in blueprint._
```

Comment:
I think `How to locally test blueprint library in your project` needs to go here right? [how-to-locally-test-blueprint-components-in-your-project](https://github.com/aimementoring/blueprint#how-to-locally-test-blueprint-components-in-your-project)

Reply:
I don't understand @bronz3beard 😬

Reply:
Hey Lara,

_"If you want to locally test a component from within your project. See_ `(this hash link doesn't work -> [How to locally test blueprint library in your project](how-to-locally-test-blueprint-library-in-your-project))' _Otherwise, read on to test your built component in blueprint."_

The hash link above didn't take me anywhere? I found the info I needed in the [commands list](https://github.com/aimementoring/blueprint#npm-commands) and this [link](https://github.com/aimementoring/blueprint#how-to-locally-test-blueprint-components-in-your-project) to get the project started locally.

I think the hash link above should link to [here](https://github.com/aimementoring/blueprint#how-to-locally-test-blueprint-components-in-your-project)

Reply:
oh! I get you! my mistake. I'll fix thaat up. thank you!